docker-compose.yml

```diff
@@ -69,12 +73,38 @@ services:
       ddd-autohaus:
         condition: service_started
 
+  dqDatabase:
+    image: mongo
+    container_name: 07_dqdatabase
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: example
+    ports:
+      - "$MONGO_PORT:$MONGO_PORT"
```

Comment:
This line gives me syntax errors

Reply:
added {}, so the synta errors should be gone